docker/sds-api/spine-directory-service/sds/sds-mock-ldap.md

```diff
@@ -1,52 +1,17 @@
 ## SDS - Spine Route Lookup component - Mock LDAP
 
-Spine Route Lookup component has a built in LDAP mocking capability
-To be able to use it, three json configuration files has to be present:
+SDS component has a built in LDAP mocking capability configurable via environment variables:
 
-- server_info.json
-- server_schema.json
-- server_entries.json
+- `SDS_MOCK_LDAP_RESPONSE` - turns the behaviour on and off (`"True"` / `"False"`)
```

Comment:
can we add these mock specific variables to the docker-compose.yml with defaults or in a override yaml file?